### PR TITLE
SQL-883: Add property to indicate connections made with Tableau

### PR DESCRIPTION
**Pull request by bucaojit**

### CODE REVIEW

File: evergreen.yml [hunk 1]
```diff
@@ -113,6 +117,19 @@ functions:
             --notary-url http://notary-service.build.10gen.cc:5000 \
             mongodb_jdbc.taco
 
+  "update plugin-version":
```
Comment (bucaojit) [inline evergreen.yml]: I have a similar change in PR 11, I will merge the change based on which is checked in first.

Comment (nbagnard) [reply evergreen.yml]: 👍 So that it will update both manifest.xml and connectionProperties.js, correct?

**Review by nbagnard — APPROVED** [review]

File: evergreen.yml (hunk 1 above)
Comment (bucaojit) [reply evergreen.yml]: Yes, that's correct.

**Review by mattChiaravalloti — APPROVED** [review]

**Comment by bucaojit** [comment]
evergreen merge

**Pull request merged**